Libraries/Components/ScrollView/ScrollView.windows.js

```diff
@@ -409,7 +409,34 @@ const ScrollView = createReactClass({
      * The default value is false.
      * @platform ios
      */
-    DEPRECATED_sendUpdatedChildFrames: PropTypes.bool,
+    DEPRECATED_sendUpdatedChildFrames: PropTypes.bool, 
+    /**
+     * Called when key down bubbles from a component that has focus.
+     *
+     * @platform windows
+     */
+    onKeyDown: PropTypes.func,
+    /**
+     * Called when key up bubbles from a component that has focus.
+     *
+     * @platform windows
+     */
+    onKeyUp: PropTypes.func,
+    /**
+     * Disables default keyboard arrow-based scrolling.
+     * 
+     * @platform windows
+     */
+    disableKeyboardBasedScrolling: PropTypes.bool,
```

Comment:
>disableKeyboardBasedScrolling [](start = 4, length = 29)

How did you decide to have this prop rather than relying on the `handledKeyDownKeys` and `handledKeyUpKeys` on descendants?

Reply:
That would be a fragile protocol:
- would assume descendants are always "WindowControl" components, but others could also be imagined
- JS side would have to keep track of "ancestor is a ScrollViewer with this flag on", or something similar.
I think the current approach is well encapsulated.

---
In reply to: [157099797](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r157099797) [](ancestors = 157099797)